### ISSUE TITLE
loss function evaluated as zero. 

### ISSUE THREAD

**Issue by jungsdao** [ACEsuit/mace#314]
During training, I'm observing weird cases of getting zero loss function value over epochs and model is not improving at all. 

[umbrella_run-123.log](https://github.com/ACEsuit/mace/files/14141717/umbrella_run-123.log)

I'm not sure what is causing this error. I'm using main branch and the latest version of mace. 
Many thanks in advance!

**Comment by ilyes319** [comment]
Are you sure you put in the right keys in the input file for xyz?

**Comment by jungsdao** [comment]
It was my mistake of using wrong key for energy and forces. Sorry for bothering with trivial mistake! Thank you for your help :)On 2. Feb 2024, at 16:48, Ilyes Batatia ***@***.***> wrote:﻿
Are you sure you put in the right keys in the input file for xyz?
